test/helpers/gas-usage.ts

```diff
@@ -13,21 +13,29 @@ export enum GasGroup {
   BULK_TRANSFER_VALIDATOR,
   BULK_TRANSFER_VALIDATOR_NON_EXISTING_POD,
   LIQUIDATE_VALIDATOR,
+  DEPOSIT,
+  WITHDRAW,
+  REGISTER_POD,
+  REACTIVATE_POD,
 }
 
 const MAX_GAS_PER_GROUP: any = {
   [GasGroup.REGISTER_OPERATOR]: 105000,
   [GasGroup.REMOVE_OPERATOR]: 45000,
   [GasGroup.REGISTER_VALIDATOR_EXISTING_POD]: 202000,
   [GasGroup.REGISTER_VALIDATOR_EXISTING_CLUSTER]: 220000,
-  [GasGroup.REGISTER_VALIDATOR_NEW_STATE]: 333000, // 313000
+  [GasGroup.REGISTER_VALIDATOR_NEW_STATE]: 319000, // 313000
   [GasGroup.REMOVE_VALIDATOR]: 120000,
   [GasGroup.TRANSFER_VALIDATOR_NEW_CLUSTER]: 400000,
   [GasGroup.TRANSFER_VALIDATOR]: 260000,
   [GasGroup.TRANSFER_VALIDATOR_NON_EXISTING_POD]: 290000,
   [GasGroup.BULK_TRANSFER_VALIDATOR]: 362000,
   [GasGroup.BULK_TRANSFER_VALIDATOR_NON_EXISTING_POD]: 379000,
-  [GasGroup.LIQUIDATE_VALIDATOR]: 101000,
+  [GasGroup.LIQUIDATE_VALIDATOR]: 120000,
+  [GasGroup.DEPOSIT]: 84000,
+  [GasGroup.WITHDRAW]: 96000,
+  [GasGroup.REGISTER_POD]: 141000,
+  [GasGroup.REACTIVATE_POD]: 251000,
```

Comment:
@vadiminc @andrew-blox Not clear why reactivate is so expensive compared to register

Reply:
@lior-blox @andrew-blox as I've removed "create pod" from the deposit func, we saved 10k for reactivepod as well